engine/schema/src/com/cloud/upgrade/dao/Upgrade41000to41100.java

```diff
@@ -52,8 +64,181 @@ public boolean supportsRollingUpgrade() {
 
     @Override
     public void performDataMigration(Connection conn) {
+        updateSystemVmTemplates(conn);
     }
 
+    @SuppressWarnings("serial")
+    private void updateSystemVmTemplates(final Connection conn) {
```

Comment:
:) we really need to convert this a an API one time

Reply:
Yes :)